examples_tests/42.FragmentShaderPathTracer/main.cpp

```diff
@@ -320,9 +324,15 @@ int main()
 		writeDescriptorSet.info = &info;
 		device->updateDescriptorSets(1u, &writeDescriptorSet, 0u, nullptr);
 	}
+	
+	// TODO: Temp Fix because of validation error: VkPhysicalDeviceLimits::nonCoherentAtomSize
+	struct alignas(64) SBasicViewParametersAligned
```

Comment:
`alignas` is not the correct solution, the correct solution is to roundup the alignment and size in the staging buffer `alloc_addr` used inside `utilities->updateBufferRangeViaStagingBuffer`

Reply:
I'm waiting for the fix :)

Reply:
@achalpandeyy ?

Reply:
Also something related to VK_WHOLE_SIZE or something?

the validation error we're trying to solve is this:
> vkFlushMappedMemoryRanges: Size in pMemRanges[0] is 0xa0, which is not a multiple of VkPhysicalDeviceLimits::nonCoherentAtomSize (0x40). The Vulkan spec states: If size is not equal to VK_WHOLE_SIZE, size must either be a multiple of VkPhysicalDeviceLimits::nonCoherentAtomSize, or offset plus size must equal the size of memory

Reply:
`updateBufferRangeViaStagingBuffer` doesn't flush your UBO buffer, it flushes the staging buffer's allocation, so VK_WHOLE_SIZE is not relevant

Reply:
done